Gemfile.lock

```diff
@@ -489,7 +489,7 @@ GEM
     parser (2.2.0.pre.5)
       ast (>= 1.1, < 3.0)
       slop (~> 3.4, >= 3.4.5)
-    payday_loans_intervention (1.4.6.251)
+    payday_loans_intervention (1.4.6.252)
```

Comment:
Hey @johnplayer1982 you have bumped it to 1.4.7 instead. The build number is not a reliable storyteller.